content/v2/openapi.yml

```diff
@@ -3931,6 +4006,12 @@ components:
       properties:
         domain:
           $ref: '#/components/schemas/Domain'
+    EventDomainLock:
+      type: object
+      description: Payload for domain.lock
```

Comment:
`domain.lock` is too generic. Note we had the same issue in the actual interactor code, and [they were renamed](https://github.com/dnsimple/dnsimple-app/pull/23326).

Please use something that is more closely associated with the notion of transfer lock, for instance:

- `domain.transfer_lock_enable`
- `domain.transfer_lock_disable`

Reply:
@weppos I have renamed the event and payload items per your suggestion.

Reply:
@AGS4NO did you use "enable" or "enable**d**" (notice the -ed form). For consistency with all the other events, we should use "enable/disable"

Reply:
Never mind, I was looking an outdated diff where it used the -ed form, but it sounds like it is now fixed.